utils/service_discovery/sd_docker_backend.py

```diff
@@ -337,8 +338,13 @@ def get_tags(self, state, c_id):
 
         return tags
 
+    def _get_container_name(self, state, c_id, tpl_var):
```

Comment:
yeah it's passed to all tag extractors by https://github.com/DataDog/dd-agent/blob/5.14.1/utils/service_discovery/sd_docker_backend.py#L489 but only used by some.